spec/intro.md

```diff
@@ -78,10 +78,6 @@ A reference to a _term_ looks like this.
 
 ### Stability Policy
```

Comment:
There's a NOTE in this section that includes a SHOULD; that ought to be fixed as well.

Reply:
I filed #1010 for that.